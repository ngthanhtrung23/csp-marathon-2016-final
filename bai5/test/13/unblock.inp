111234
55.234
60023B
6.77.B
..8999
..8AAA
