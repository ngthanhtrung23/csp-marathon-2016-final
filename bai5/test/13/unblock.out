22
5 R 1
6 U 1
7 L 2
2 D 1
1 R 1
6 U 1
0 L 1
8 U 2
9 L 3
A L 3
2 D 2
3 D 3
5 R 2
B D 1
4 D 1
1 R 2
8 U 2
0 R 3
8 D 1
1 L 1
4 U 1
0 R 1

